include/data/FluidJSON.hpp

```diff
@@ -186,6 +187,31 @@ void from_json(const nlohmann::json &j, KMeans &kmeans) {
   kmeans.setMeans(means);
 }
 
+// SKMeans
+void to_json(nlohmann::json &j, const SKMeans &skmeans) {
+  RealMatrix means(skmeans.getK(), skmeans.dims());
+  skmeans.getMeans(means);
+  j["means"] = RealMatrixView(means);
+  j["rows"] = means.rows();
+  j["cols"] = means.cols();
+}
+
+bool check_json(const nlohmann::json &j, const SKMeans &) {
+  return fluid::check_json(j,
+    {"rows", "cols", "means"},
+    {JSONTypes::NUMBER, JSONTypes::NUMBER,JSONTypes::ARRAY}
+  );
+}
+
+void from_json(const nlohmann::json &j, SKMeans &skmeans) {
+  index rows = j.at("rows");
+  index cols = j.at("cols");
```

Comment:
```suggestion
  index rows = j.at("rows").get<index>();
  index cols = j.at("cols").get<index>();
```
The implicit conversion in `at()` is now not recommended by the JSON author, especially in `from_json` https://nicedoc.io/nlohmann/json#implicit-conversions